### DISCUSSION TITLE
When will a new version be released?

### DISCUSSION THREAD

**Discussion by jpneonascimento** [phpredis/phpredis#2474] [General]
Guys, I would like to know if there is an estimate of when a new version will be released considering this fix that was accepted.

- https://github.com/phpredis/phpredis/pull/2459

We are having problems connecting to redis in cluster mode and want to use this fix.

**Comment by MagnumCortez** [comment]
I have the same doubt

**Comment by michael-grunder** [comment]
We can look to put together a release in the near future.

Since there are likely a fair number of changes since our last release it's probably prudent to put out a release candidate first.

As for #2459, I am concerned that this is a breaking change.  We might be able to get away with it if we consider the previous behavior a bug although I'm sure this will break someone's code.

**Comment by cyppe** [comment]
Nice, also looking forward for a RC soon.  Currently compiling latest dev so no big deal but it's more handy to have official release.

**Comment by wcoppens** [comment]
Looking forward to that RC! 🚀

**Comment by kalifg** [comment]
Also looking forward to it!

**Comment by kiettran** [comment]
Looking forward to the new release! 🔥

**Comment by pkrijgsman** [comment]
It would be very nice to have the new release soon 💪

**Comment by michael-grunder** [comment]
OK I can put together a release.  Not sure whether we need a release candidate but it might be prudent.

**Comment by mfn** [comment]
I've a question regarding a changelog entry -> maybe I got it wrong but look at this:
- https://github.com/phpredis/phpredis/blob/develop/CHANGELOG.md#610rc1---2024-08-04-github-pecl lists this entry
  > Fix memory leak and segfault in Redis::exec [362e1141](https://github.com/phpredis/phpredis/commit/362e1141) ([Pavlo Yatsukhnenko](https://github.com/yatsukhnenko))
  
  and also the commit itself shows this correctly -> https://github.com/phpredis/phpredis/commit/362e1141
    
  ![image](https://github.com/user-attachments/assets/c03128da-e58f-4b97-865b-a88572c06552)
- but I know for a fact, this was already release with 6.0.1 that very day and it's also listed at https://github.com/phpredis/phpredis/blob/develop/CHANGELOG.md#601---2023-09-23-github-pecl
  > Fix memory leak and segfault in Redis::exec
[362e1141](https://github.com/phpredis/phpredis/commit/362e1141)
([Pavlo Yatsukhnenko](https://github.com/yatsukhnenko)),
([Markus Podar](https://github.com/mfn))

So: the _latest_ cangelog entry and the commit itself even suggests that this was part of 6.1.0-RC1 first, but it was already released with 6.0.1 _almost_ a year ago.

Any idea why this discrepancy?

Thanks!

**Reply by michael-grunder** [reply]
I might have just brought it in twice. I did the changelog manually. I can fix this before the full release.
